CHANGELOG.md

```diff
@@ -2,6 +2,31 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.79.0](https://github.com/sonata-project/SonataAdminBundle/compare/3.78.1...3.79.0) - 2020-11-09
+### Added
+- [[#6565](https://github.com/sonata-project/SonataAdminBundle/pull/6565)] Added `collection_by_reference` option for AdminType ([@VincentLanglet](https://github.com/VincentLanglet))
+- [[#6556](https://github.com/sonata-project/SonataAdminBundle/pull/6556)] Added `Sonata\AdminBundle\Templating\AbstractTemplateRegistry` ([@wbloszyk](https://github.com/wbloszyk))
+- [[#6556](https://github.com/sonata-project/SonataAdminBundle/pull/6556)] Added `Sonata\AdminBundle\Templating\MutableTemplateRegistry` ([@wbloszyk](https://github.com/wbloszyk))
+- [[#6556](https://github.com/sonata-project/SonataAdminBundle/pull/6556)] Added `Sonata\AdminBundle\Templating\MutableTemplateRegistryAwareInterface` ([@wbloszyk](https://github.com/wbloszyk))
+- [[#6463](https://github.com/sonata-project/SonataAdminBundle/pull/6463)] Added DataSourceInterface ([@VincentLanglet](https://github.com/VincentLanglet))
+
+### Changed
+- [[#6565](https://github.com/sonata-project/SonataAdminBundle/pull/6565)] `collection_by_reference` is used instead of `by_reference` in AdminType in order to know which call should be used between `ObjectManipulator::addInstance` and `ObjectManipulator::setObject`. ([@VincentLanglet](https://github.com/VincentLanglet))
```

Comment:
```suggestion
- [[#6565](https://github.com/sonata-project/SonataAdminBundle/pull/6565)] `collection_by_reference` is used instead of `by_reference` in `AdminType` in order to know which call should be used between `ObjectManipulator::addInstance()` and `ObjectManipulator::setObject()` ([@VincentLanglet](https://github.com/VincentLanglet))
```